src/components/settings/ImportAllDialog/index.tsx

```diff
@@ -0,0 +1,250 @@
+        }`,
+      }),
+    )
+
+    trackEvent({
+      ...SETTINGS_EVENTS.DATA.IMPORT_ALL,
+      label: `Added Safes: ${addedSafesCount}, Addressbook entries: ${addressBookEntriesCount}`,
+    })
+
+    handleClose()
+  }
+
+  return (
+    <ModalDialog open onClose={handleClose} dialogTitle="Data Import" hideChainIndicator>
+      <DialogContent>
+        <div>
+          {fileName ? (
+            <Box>
+              <>
+                <Grid container direction="column" gap={1}>
+                  <Grid container gap={1} display="flex" alignItems="center">
+                    <Grid item xs={1}>
+                      <SvgIcon
+                        component={FileIcon}
+                        inheritViewBox
+                        fontSize="small"
+                        color="primary"
+                        sx={{ fill: 'none' }}
+                      />
+                    </Grid>
+                    <Grid item xs={7}>
+                      {fileName}
+                    </Grid>
+
+                    <Grid item xs display="flex" justifyContent="flex-end">
+                      <IconButton onClick={onRemove}>
+                        <HighlightOffIcon color="primary" width={16} height={16} />
+                      </IconButton>
+                    </Grid>
+                  </Grid>
+                  <Grid item xs={12} display="flex" justifyContent="flex-start">
+                    <div className={css.vertical_line} />
+                  </Grid>
+                  <>
+                    {addressBook && (
+                      <Grid container display="flex" gap={1} alignItems="center">
+                        <Grid item xs={1}>
+                          <SvgIcon
+                            component={FileIcon}
+                            inheritViewBox
+                            fontSize="small"
+                            color="border"
+                            sx={{ fill: 'none' }}
+                          />
+                        </Grid>
+                        <Grid item xs>
+                          <Typography>
+                            Found <strong>{addressBookEntriesCount} Address book</strong> entries for{' '}
+                            <strong>{Object.keys(addressBook).length} chains</strong>
+                          </Typography>
+                        </Grid>
+                      </Grid>
+                    )}
+                    {addedSafes && (
+                      <Grid container display="flex" gap={1} alignItems="center">
+                        <Grid item xs={1}>
+                          <SvgIcon
+                            component={FileIcon}
+                            inheritViewBox
+                            fontSize="small"
+                            color="border"
+                            sx={{ fill: 'none' }}
+                          />
```

Comment:
This is repeated three times. Let's move it to either it's own component or a variable.